client/src/dummyData/usersGithubUsers.js

```diff
@@ -0,0 +1,38 @@
+export const usersGithubUsers = [
+    {
+        username: 'user1',
+        repositories_count: 324
+    },
+    {
+        username: 'user2',
+        repositories_count: 535
+    },
+    {
+        username: 'user3',
+        repositories_count: 54567
+    },
+    {
+        username: 'user5',
+        repositories_count: 4235534
+    },
+    {
+        username: 'user6',
+        repositories_count: 323
+    },
+    {
+        username: 'user7',
+        repositories_count: 3
+    },
+    {
+        username: 'user8',
+        repositories_count: 1
+    },
+    {
+        username: 'user9',
+        repositories_count: 3
+    },
+    {
+        username: 'user10',
+        repositories_count: 69
+    },
+];
```

Comment:
Like i mentioned before, i would add all parameters, we are going to receive from backend later.

Reply:
On next sync meeting we are going (I hope so)  to create our API template. Then we will know exactly what we are going to recive